docs/getting-started/auto-suggest.md

```diff
@@ -0,0 +1,212 @@
+---
+layout: default
+title: Auto-Suggest
+nav_order: 5
+parent: Getting Started
+has_toc: false
+---
+
+# Intelligent Auto-Suggest
+
+{: .no_toc }
+
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+1. TOC
+{:toc}
+</details>
+
+---
+
+## Introduction
+
+> Introduced in Tasks 1.9.0.
+
+The [Priorities]({{ site.baseurl }}{% link getting-started/priority.md %}), [Dates]({{ site.baseurl }}{% link getting-started/dates.md %}) and [Recurring Tasks]({{ site.baseurl }}{% link getting-started/recurring-tasks.md %}) pages show various emojis and special phrases that the Tasks plugin recognises, when searching for tasks.
+
+If you prefer to type your tasks, instead of using a dialog, there is now an intelligent auto-suggest completion mechanism that does a
+lot of the typing of emojis and dates for you.
+
+It is particularly powerful when creating and editing tasks on mobile phones.
+
+### Video Demo
+
+It is perhaps best understood by watching a [video of it in action](https://user-images.githubusercontent.com/10722656/175102574-78b0f851-cc48-4255-a40e-d3036bec5bb6.gif).
+
+### Walk Through
+
+Here is a more detailed walk through of the creation of a new task, which can be done entirely using the keyboard if you wish.
+
+1. As you are type a task, the auto-complete menu pops up to show some common options:
```

Comment:
1. I believe "you are typing"?
2. It's important to mention here that the auto-complete pops up only after the line can be recognized as a task, i.e. it contains a bullet with a checkbox (`- [ ]`) **and** the global filter if any.

Reply:
#2 is actually mentioned below, but I suggest mentioning it here too for the benefit of the impatient :)

Reply:
1. Thank you: changed to:

    > As you are typing a task, the auto-suggest menu pops up to show some common options:

2. Added:

    **Note**: the auto-suggest menu pops up only if the cursor is in a line that is recognized as a task, that is, the line contains:
    
     - a bullet with a checkbox (`- [ ]` or `* [ ]`)
     - and the global filter (if any)

Also added, at the end of the "Details" list:

- It currently pops up when editing completed tasks. This may be changed in future.